runtime/syntax/python.vim

```diff
@@ -77,13 +77,14 @@ endif
 "
 " The list can be checked using:
 "
-" python3 -c 'import keyword, pprint; pprint.pprint(keyword.kwlist, compact=True)'
+" python3 -c 'import keyword, pprint; pprint.pprint(keyword.kwlist, compact=True);\
+"" ;  pprint.pprint(keyword.softkwlist, compact=True)'
```

Comment:
Is this second `;` needed?
It seems it can be removed because there's on at the end of the previous line.

Reply:
Actually that is a kind of hacky attempt to make copy and pasting this easier: `python3 -c 'import keyword, pprint; pprint.pprint(keyword.kwlist, compact=True);\
"" ;  pprint.pprint(keyword.softkwlist, compact=True)'` works because Python accepts the empty string as no-op command. I admit that it looks wrong and will confuse most readers, therefore I'll probably remove it. Should I go with a very long line or just leave the task of removing the commenting `"` to the user?

Or maybe a slightly longer line would be ideal, I think I'd prefer that but I'm not sure which style you prefer
`" python3 -c 'import keyword, pprint; pprint.pprint(keyword.kwlist + keyword.softkwlist, compact=True)'` (104 characters)

Reply:
I force-pushed a new commit with the last option. I think it's ideal if you don't mind the slightly longer line, because it's not confusing but still easy to copy and paste.

Reply:
This is the best choice. I agree.
Tip for future PR updates: [–force considered harmful; understanding git’s –force-with-lease](https://blog.developer.atlassian.com/force-with-lease/)
🙂